app/helpers/application_helper.rb

```diff
@@ -8,4 +8,21 @@ def stylesheet(*files)
     content_for(:head) { stylesheet_link_tag(*files) }
   end
 
+  def list?
```

Comment:
Why need `!!` convertions?
The style guide say avoid that:
https://github.com/rubocop-hq/ruby-style-guide#no-bang-bang

Reply:
Because of the safe navigation, `current_user.view_settings&.list?` will return `nil` if user doesn't have `view_setting` or it will return `true` or `false` according to `list?`. So here using `nil?` is not an option, and I think returning `nil` and using as `false` is not so nice either.